script/ci/restore-staging-baseline.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Restore baseline data for staging in case a pull request messed with
+# the database.
+
+set -e
+
+: ${DB_USER='ofn_user'}
```

Comment:
TIL `:` in bash is an alias of `true` 
For an explanation of what is happening here : https://stackoverflow.com/a/4892546

Reply:
I assume the intention here is to set a default value none is set, and from what I understand we need `:=` here.

Also, I like that this is a nice and succinct way to specify a default value, but I only come across it once a year and need to look it up every time, maybe a comment will help.

```suggestion
# Load default values if not already set
: ${DB_USER:='ofn_user'}
```

Reply:
Good idea to add the comment, I'll do that.

The colon is not necessary for the assignment here. There's a slight difference. If DB_USER is already set to a string then that's used in both cases. If the variable is not set at all then the default value is assigned in both cases. The difference comes in with an _assigned null_ value. My version `=` would then still use null and your version `:=` would still assign the default value 'ofn_user'.

Since we are not assigning the variable to null, it doesn't matter. 

* https://unix.stackexchange.com/a/122848

Reply:
Thanks for explaining. But Bash parameter expansion is too quirky, and I don't place any expectation on myself to remember that.. So thanks for adding the comment :)